src/deepsparse/loggers/prometheus_logger.py

```diff
@@ -98,18 +116,22 @@ def log(self, identifier: str, value: Any, category: MetricCategories):
         :param category: The metric category that the log belongs to
         """
         prometheus_metric = self._get_prometheus_metric(identifier, category)
+        if prometheus_metric is None:
+            return
         prometheus_metric.observe(self._validate(value))
         self._export_metrics_to_textfile()
 
     def _get_prometheus_metric(
         self, identifier: str, category: MetricCategories
-    ) -> Summary:
+    ) -> Union[None, Histogram, Gauge, Summary, Counter]:
```

Comment:
if we're going to be using this a lot for our supported prometheus metrics, I'd suggest maybe adding an alias for this ie
```
PrometheusMetric = Union[Histogram, Gauge, Summary, Counter]
```
and update to `Optional[PROMETHEUS_METRIC]